exporters/FusionRobotExporter/FusionRobotExporter.vcxproj

```diff
@@ -70,7 +70,9 @@
       <AdditionalDependencies>core.lib;fusion.lib;cam.lib;kernel32.lib;user32.lib;gdi32.lib;winspool.lib;comdlg32.lib;advapi32.lib;shell32.lib;ole32.lib;oleaut32.lib;uuid.lib;odbc32.lib;odbccp32.lib;Rpcrt4.lib;vhacd.lib;%(AdditionalDependencies)</AdditionalDependencies>
     </Link>
     <PostBuildEvent>
-      <Command>xcopy /R /Y "$(SolutionDir)$(Configuration)\$(ProjectName).dll" "$(SolutionDir)"</Command>
+      <Command>xcopy /E /I /Y /S "$(SolutionDir)palette" "$(SolutionDir)$(Configuration)\palette"
+xcopy /E /I /Y /S "$(SolutionDir)Resources" "$(SolutionDir)$(Configuration)\Resources"
+xcopy /Y "$(SolutionDir)$(ProjectName).manifest" "$(SolutionDir)$(Configuration)"</Command>
```

Comment:
This is where the palette folder, resources folder, and .manifest file are copied.